pkg/resources/reconciler.go

```diff
@@ -111,18 +112,17 @@ func (r *Reconciler) ReconcileNamespace(ctx context.Context, namespace string, i
 
 		return integreatlyv1alpha1.PhaseCompleted, nil
 	}
-	// ns exists so check it is our namespace
-	if !IsOwnedBy(ns, inst) && ns.Status.Phase != corev1.NamespaceTerminating {
-		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("existing namespace found with name %v but it is not owned by the integreatly installation and it isn't being deleted", ns.Name)
+
+	PrepareObject(ns, inst)
+	if err := client.Update(ctx, ns); err != nil {
```

Comment:
I am not sure why this was changed. With new behavior, any "left-over" namespaces from a previous Installation instance will be re-purposed for a new installation. This will happen because we will override "ownership" label instead of failing as we did in the past.
btw: if we keep this change, we no longer need `IsOwnedBy` function.

Reply:
This was removed because the installation keeps failing on "namespace exists but is not owned by us", and as we're using a reserved namespace prefix, I don't think this worry is too pressing.